.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,79 @@
+name: Docker
+
+on:
+  push:
+    # Publish `master` as Docker `latest` image.
+    branches:
+      - master
+
+    # Publish `v1.2.3` tags as releases.
+    tags:
+      - v*
+
+  # Run tests for any PRs.
+  pull_request:
```

Comment:
Is this meant to be empty? I'd like docker to run on PRs as well

Reply:
It appears to be running the tests on this PR, I cannot remember why thats there, I just took the default docker publish action

Reply:
Okay, I guess it works if it's empty lol

Reply:
It probably is supposed to be empty unless you want it to publish, but again, not sure.